### PR TITLE
HttpClient 4.x upgrade

### PR DESCRIPTION
**Pull request by TheAntimist**
For merging, v1.0 into master.

### CODE REVIEW

**Comment by TheAntimist** [comment]
Looks like CI failed due to Javadoc errors. Should fix that before release.

**Comment by TheAntimist** [comment]
Fixed those javadoc errors, and currently pushing to maven central as 1.0.0-rc.1

**Comment by TheAntimist** [comment]
@ioggstream Any update?

**Comment by ioggstream** [comment]
Will check tomorrow. Sorry.

**Comment by TheAntimist** [comment]
No worries, take your time

File: src/main/java/com/github/caldav4j/CalDAVConstants.java [hunk 1]
```diff
@@ -14,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.osaf.caldav4j;
+package com.github.caldav4j;
```
Comment (ioggstream) [inline src/main/java/com/github/caldav4j/CalDAVConstants.java]: Is that aligned to maven central?

File: src/main/java/com/github/caldav4j/methods/HttpPropFindMethod.java [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+	 * Convenience method to return the Calendar Display Name.
+	 * @param urlPath Location of the CalendarResource
+	 * @param httpResponse Response Object for the request.
+	 * @return Display Name as string
+	 */
+	public String getDisplayName(HttpResponse httpResponse, String urlPath) {
+		DavProperty p = getDavProperty(httpResponse, urlPath, DavPropertyName.DISPLAYNAME);
+		if (p != null && p.getValue() != null) {
+			return p.getValue().toString();
+		} else {
```
Comment (ioggstream) [inline src/main/java/com/github/caldav4j/methods/HttpPropFindMethod.java]: In this simple cases I'd use:

if () {
  return
}
return "";

File: src/main/java/com/github/caldav4j/CalDAVConstants.java (hunk 1 above)
Comment (TheAntimist) [reply src/main/java/com/github/caldav4j/CalDAVConstants.java]: Yes

File: src/main/java/com/github/caldav4j/methods/HttpPropFindMethod.java (hunk 2 above)
Comment (TheAntimist) [reply src/main/java/com/github/caldav4j/methods/HttpPropFindMethod.java]: Sure, I'll handle it.

**Comment by TheAntimist** [comment]
The changes, sound good to me, will handle it over the weekend.

**Comment by TheAntimist** [comment]
Fixed all those mentioned, anything else, you want me to handle?

**Comment by ioggstream** [comment]
@TheAntimist everything fine. Did you already merge this, right?

**Comment by TheAntimist** [comment]
@ioggstream The merge is actually still pending. Will go through it, now.

**Pull request merged**